playerbot/strategy/paladin/HolyPaladinStrategy.cpp

```diff
@@ -46,6 +46,10 @@ void HolyPaladinStrategy::InitCombatTriggers(std::list<TriggerNode*>& triggers)
     triggers.push_back(new TriggerNode(
         "enemy is close",
         NextAction::array(0, new NextAction("judgement", ACTION_NORMAL), NULL)));
+
+    triggers.push_back(new TriggerNode(
+        "blessing of sacrifice",
+        NextAction::array(0, new NextAction("blessing of sacrifice", ACTION_MEDIUM_HEAL + 8), NULL)));
```

Comment:
the triggers are sorted from higher to lower relevance so should be moved up and relevancy should be ACTION_MEDIUM_HEAL + 7

Reply:
Sorted